### PR TITLE
build.xml added to the sample CarbonSaasApp

### PR DESCRIPTION
**Pull request by Manuri**

### CODE REVIEW

File: modules/samples/JavaEE-TomEE/build.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```
Comment (callkalpa) [inline modules/samples/JavaEE-TomEE/build.xml]: We need to add license header

File: modules/samples/Jaxws-Jaxrs/build.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```
Comment (callkalpa) [inline modules/samples/Jaxws-Jaxrs/build.xml]: We need to add license header

File: modules/samples/SSOValveSample/build.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```
Comment (callkalpa) [inline modules/samples/SSOValveSample/build.xml]: We need to add license header

File: modules/samples/JavaEE-TomEE/build.xml (hunk 1 above)
Comment (Manuri) [inline modules/samples/JavaEE-TomEE/build.xml]: Fixed with https://github.com/wso2/product-as/pull/346

File: modules/samples/Jaxws-Jaxrs/build.xml (hunk 2 above)
Comment (Manuri) [inline modules/samples/Jaxws-Jaxrs/build.xml]: Fixed with https://github.com/wso2/product-as/pull/346

File: modules/samples/SSOValveSample/build.xml (hunk 3 above)
Comment (Manuri) [inline modules/samples/SSOValveSample/build.xml]: Fixed with https://github.com/wso2/product-as/pull/346

**Pull request merged**